### PR TITLE
Simplify capabilities

### PR DESCRIPTION
**Pull request by jdillon**
simplifies packages, extracts discrete api into nexus-capability, moves impls into nexus-core.

related:
https://github.com/sonatype/nexus-pro/pull/556
https://github.com/sonatype/nexus-bundles/pull/149

### CODE REVIEW

**Comment by jdillon** [comment]
http://bamboo.s/browse/NX3-OSSF376-1 :white_check_mark:

File: plugins/capabilities/nexus-capabilities-testsuite-plugin/src/main/java/org/sonatype/nexus/plugins/capabilities/test/helper/RepositoryIsInServiceCapability.java [hunk 1]
```diff
@@ -57,13 +63,13 @@ public String get() {
...
   public Condition validityCondition() {
-    return conditions.repository().repositoryExists(new RepositoryName()
-    {
-      @Override
-      public String get() {
-        return context().properties().get(REPOSITORY);
-      }
-    }
+    return repositoryConditions.repositoryExists(new RepositoryName()
+                                                 {
```
Comment (mcculls) [inline plugins/capabilities/nexus-capabilities-testsuite-plugin/src/main/java/org/sonatype/nexus/plugins/capabilities/test/helper/RepositoryIsInServiceCapability.java]: odd formatting...

**Comment by mcculls** [comment]
+1

File: plugins/capabilities/nexus-capabilities-testsuite-plugin/src/main/java/org/sonatype/nexus/plugins/capabilities/test/helper/RepositoryIsInServiceCapability.java (hunk 1 above)
Comment (jdillon) [inline plugins/capabilities/nexus-capabilities-testsuite-plugin/src/main/java/org/sonatype/nexus/plugins/capabilities/test/helper/RepositoryIsInServiceCapability.java]: very, will fix

**Comment by adreghiciu** [comment]
+1

**Comment by cstamas** [comment]
+1

**Comment by jdillon** [comment]
http://bamboo.s/browse/NX3-OSSF376-2 :white_check_mark:

**Comment by jdillon** [comment]
manually merged

**Pull request closed without merging**